test/controllers/offlinePayment.controller.test.js

```diff
@@ -208,7 +208,7 @@ describe('Offline Payment Controller', () => {
       context('when model id empty', () => {
```

Comment:
Lets add a similar test for `vasRequestId`.

```suggestion
      context('when offer id empty', () => {
```

Also, let's add another test to ensure that an error is returned when the associated model is not given based on the model type (as advised on the Joi implementation).

Something like this:

```js
 context('when offer id is not given', () => {
        it('retruns error', (done) => {
          const offlinePayment = OfflinePaymentFactory.build({
            model: { type: MODEL.OFFER },
            bankId: bank._id,
          });
          request()
            .post('/api/v1/offline-payment')
            .set('authorization', userToken)
            .send(offlinePayment)
            .end((err, res) => {
              expect(res).to.have.status(412);
              expect(res.body.success).to.be.eql(false);
              expect(res.body.error).to.be.eql('"Offer id" is required');
              done();
            });
        });
      });

```

Similar tests should be added for VasRequestId too. Also, similar updates should be made in `transaction.controller.test.js` for both models.